mne/viz/topomap.py

```diff
@@ -2806,3 +2807,206 @@ def plot_bridged_electrodes(info, bridged_idx, ed_matrix, title=None,
         cax = fig.colorbar(im)
         cax.set_label(r'Electrical Distance ($\mu$$V^2$)')
     return fig
+
+
+def plot_ch_adjacency(info, adjacency, ch_names, kind='2d', edit=False):
+    """Plot channel adjacency.
+
+    Parameters
+    ----------
+    info : instance of Info
+        Info object with channel locations.
+    adjacency : array
+        Array of channels x channels shape. Defines which channels are adjacent
+        to each other. Note that if you edit adjacencies
+        (via ``edit=True``), this array will be modified in place.
+    ch_names : list of str
+        Names of successive channels in the ``adjacency`` matrix.
+    kind : str
+        How to plot the adjacency. Can be either ``'3d'`` or ``'2d'``.
+    edit : bool
+        Whether to allow interactive editing of the adjacency matrix via
+        clicking respective channel pairs. Once clicked, the channel is
+        "activated" and turns green. Clicking on another channel adds or
+        removes adjacency relation between the activated and newly clicked
+        channel (depending on whether the channels are already adjacent or
+        not); the newly clicked channel now becomes activated. Clicking on
+        an activated channel deactivates it. Editing is currently only
+        supported for ``kind='2d'``.
```

Comment:
when you click outside of any electrode i think it should remove the green segments

Reply:
I can change it, but it may be a bit frustrating if someone misses the desired channel (to add/remove connection) from time to time and it deselects his current one.

Reply:
@agramfort did you pull the changes and try it yourself? I find the user experience of the current interface quite nice.

Reply:
I'm also not sure how it should be implemented. Currently only pick events are handled. To support clicking outside I'd have to add another mouse event handler. But I think then if one clicks on the channel both events would fire, so the second mouse event would have to somehow detect if the first event was activated?

Reply:
I think we should keep the current user interface -- if trying the code in real life shows us that indeed we should change it because there are pain points, let's do a follow up PR.